code/modules/jobs/job_types/!crashsite/ship_commander.dm

```diff
@@ -0,0 +1,48 @@
+/datum/job/ship_commander
+	title = "Ship Commander"
+	department_head = list("Syndicate Command")
+	faction = "Syndicate"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the syndicate"
+	selection_color = "#ab1f18"
+	exp_requirements = 60
+	exp_type = EXP_TYPE_SYNDICATE
+	maptype = "crashsite"
+	trusted_only = TRUE
```

Comment:
Is this meant to be whitelisted only?
Remember our policy about whitelisting roles:
If griefing as it ruins the entire round for everyone, whitelist it.

I'm leaving this up to you though